test/complex.jl

```diff
@@ -172,6 +176,55 @@ end
     end
 end
 
+@testset "complex ldexp" begin
+    @testset "Complex{$T}" for F in (Float16,Float32,Float64)
```

Comment:
I'm not sure it's worth just copying the tests from the the float versions of `ldexp` wholesale, given that the implementations are tightly coupled.  It might be better just to test a few examples, given that complex ldexp inherits its behavior directly from the float version.